.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
As mostly a vim user, I'm not too familiar with how vscode settings are distributed with source packages. Is version-controlling `settings.json` generally good practice?

Reply:
This is an excellent question!

In earnest, I think it's really at your discretion...

I like to keep the .vscode settings in the repo so that they'll be picked up across platforms and by any other developers using VSCode, that said, there's no inherent NEED for them, and could just as easily be lobbed into the `.gitignore` file. I find it generally helpful to keep them around just like any other development tooling files (like .`.gitignore`, or `.pre-comit-config.yaml`, or anything else).

Reply:
Gotcha. I'm OK with leaving it in for now, at least.